ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/hardware-counters.j2

```diff
@@ -1,10 +1,13 @@
-{% if hardware_counters is defined and hardware_counters is not none %}
-{%     if hardware_counters.features is defined and hardware_counters.features is not none %}
+{# eos - Hardware counters #}
+{% if hardware_counters is arista.avd.defined %}
```

Comment:
Both if statement can be merged as `arista.avd.defined` does a deep test

Reply:
line 2 is redundant and should be removed